src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -706,6 +740,27 @@ export function GeneralEntityRoot<T extends BaseEntity>({
           loading={isGridLoading}
           size="small"
           tableConfig={tableConfig}
+          // @ts-ignore
```

Comment:
Why is this needed?

Reply:
I'll have a look at it again, but I fear the typing in antd is not correct.

Reply:
See 45255f0.

Not sure why, but the component extended the `HTMLDivElement` type and this interfered with the `onChange` type of the table.